api/app/controllers/api/v1/projects_controller.rb

```diff
@@ -27,6 +27,7 @@ def index
       def show
         @scope_for_projects = Project.friendly
         @project = load_project
+        authorize_action_for @project if @project.draft
```

Comment:
Can you help me understand what's happening here? This doesn't make sense to me. It seems to me that we should always authorize the project show method and handle any conditional logic in the project authorizer. If the project is set to draft and the user is not an admin, then the user is NOT authorized to show the project.